### DISCUSSION TITLE
Automatic Save

### DISCUSSION THREAD

**Discussion by tholum** [cline/cline#381] [Feature Requests]
I would love to be able to give it a task, and have it automatically save ( Like we have the option to grant permisions to automaticly read )

**Comment by xvweirong** [comment]
Especially when I use it to create things in a domain that I am not familiar with. The approval feature doesn't make sense in that scenario.

**Reply by TomLucidor** [reply]
Wouldn't you need a rollback feature with this at well tho if something goes wrong?

**Reply by armper** [reply]
I was thinking about this problem and the solution could be just do what a human would do: use git in order to roll back to the previous version. Cline and ai has no problem making commits. I do it all the time because I'm extra lazy :)

**Comment by dcbartlett** [comment]
Cline functionality supports this today.  Closing.

**Reply by 619** [reply]
>Says feature already exists
>Doesn't tell us where it is

Is this stack overflow?
